src/d3a/d3a_core/sim_results/area_statistics.py

```diff
@@ -592,3 +591,59 @@ def _convert_output_format(self, price_energy, csv_output, redis_output):
                 } for timeslot, trades in trade_rates.items()
             ]
             redis_output[node.uuid] = deepcopy(csv_output[node.name])
+
+
+class AreaThroughputStats:
```

Comment:
Better to add this class to its own module / file. The `area_statistics` module is already big without this new class, and it would be great to start reducing its size.

Reply:
Agreed! Doing so.